tests/test_fieldset.py

```diff
@@ -287,13 +288,16 @@ def test_fieldset_samegrids_from_file(tmpdir, filename='test_subsets'):
     files = {'U': ufiles, 'V': vfiles}
     variables = {'U': 'vozocrtx', 'V': 'vomecrty'}
     dimensions = {'lon': 'nav_lon', 'lat': 'nav_lat'}
-    chs = 'auto'
-    fieldset = FieldSet.from_netcdf(files, variables, dimensions, timestamps=timestamps, allow_time_extrapolation=True, field_chunksize=chs)
+    fieldset = FieldSet.from_netcdf(files, variables, dimensions, timestamps=timestamps, allow_time_extrapolation=True, field_chunksize=field_chunksize)
 
-    assert fieldset.gridset.size == 1
-    assert fieldset.U.grid == fieldset.V.grid
-    assert fieldset.U.grid.master_chunksize == fieldset.V.grid.master_chunksize
-    assert fieldset.U.field_chunksize == fieldset.V.field_chunksize
+    if field_chunksize == 'auto':
+        assert fieldset.gridset.size == 2
+        assert fieldset.U.grid != fieldset.V.grid
+    else:
+        assert fieldset.gridset.size == 1
+        assert fieldset.U.grid == fieldset.V.grid
+        assert fieldset.U.grid.master_chunksize == fieldset.V.grid.master_chunksize
```

Comment:
isn't this completely verbose ? if the grids are the same, it means it's the same object, so this assertion is entirely redundant.